packages/atomic/ws.config.cjs

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  rewrite: [
+    {from: '/headless/v([.0-9]*)/(.*)', to: '/headless/$2'},
+    {from: '/bueno/v([.0-9]*)/(.*)', to: '/bueno/$2'},
+  ],
+  directory: 'dist-storybook',
+  port: 4400,
+};
```

Comment:
Can you explain this ? What are these changes to atomic ? What were the problems ?

Reply:
Since we copy the headless/bueno builds without the version numbers, we need some rewrite rules so that headless/v*/* goes to headless/*